src/test/java/ca/corefacility/bioinformatics/irida/ria/integration/pages/announcements/AnnouncementDashboardPage.java

```diff
@@ -27,12 +27,15 @@ public void viewReadAnnouncements() {
     }
 
     public List<WebElement> getCurrentUnreadAnnouncements() {
-        return driver.findElements(By.cssSelector(".announcement-item"));
+        return driver.findElements(By.cssSelector(".t-announcement-item"));
```

Comment:
You can also use just `By.className("t-announcement-item")` - again just an FYI of whats available

Reply:
Thanks for letting me know!